### PR TITLE
Issue with caret displaying on select teams in firefox

### PR DESCRIPTION
**Pull request by matthewgaulin**
In Firefox, these drop downs were not displaying the caret

### CODE REVIEW

File: src/client/app/teams/select-teams.component.html [hunk 1]
```diff
@@ -23,10 +23,10 @@
 	<div *ngIf="showTags" class="col-md-6">
 		<div style="padding-left:0;" class="col-md-12 pull-left btn-group" dropdown>
 			<button style="text-align: left;" type="button" class="form-control input-sm dropdown-toggle hide-overflow" dropdownToggle>
```
Comment (reblace) [inline src/client/app/teams/select-teams.component.html]: This is not a question that should hold up this fix, but is there a reason we aren't using a combobox here?

Comment (matthewgaulin) [reply src/client/app/teams/select-teams.component.html]: I was thinking the same thing.  Maybe there could be a follow on issue to address this in a better way.

Comment (reblace) [reply src/client/app/teams/select-teams.component.html]: You want to add it to the "style consistency" ticket?

Comment (matthewgaulin) [reply src/client/app/teams/select-teams.component.html]: Added

**Review by ymao1 — APPROVED** [review]

**Comment by matthewgaulin** [comment]
Might be a cleaner way to do this looking into it now.

**Comment by matthewgaulin** [comment]
Added small change to better deal with overflow, didnt really end up being cleaner.

**Review by mheir — APPROVED** [review]

**Pull request merged**